scripts/node/twig_functions.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Install Pattern Lab.
+ */
+const shell = require("shelljs");
```

Comment:
Nitpick, but this is using double quotes while the other require statements are using single.